data/sql/updates/pending_db_world/rev_1609100844918098600.sql

```diff
@@ -0,0 +1,130 @@
+INSERT INTO `version_db_world` (`sql_rev`) VALUES ('1609100844918098600');
+
+UPDATE `creature` SET `MovementType` = 2 WHERE `guid` = 85221;
+
+DELETE FROM `creature` WHERE (`id` IN (25258, 25261, 25259) AND `guid` IN (85221, 85222, 85226));
+INSERT INTO `creature` (`guid`, `id`, `map`, `zoneId`, `areaId`, `spawnMask`, `phaseMask`, `modelid`, `equipment_id`, `position_x`, `position_y`, `position_z`, `orientation`, `spawntimesecs`, `wander_distance`, `currentwaypoint`, `curhealth`, `curmana`, `MovementType`, `npcflag`, `unit_flags`, `dynamicflags`, `ScriptName`, `VerifiedBuild`) VALUES
+(85221, 25258, 571, 0, 0, 1, 1, 23031, 0, 2280.15, 5179.14, 11.423, 4.38078, 1200, 0, 0, 1, 0, 2, 0, 0, 0, '', 0),
+(85222, 25259, 571, 0, 0, 1, 1, 23031, 0, 2282.03, 5179.58, 11.423, 3.76991, 1200, 0, 0, 1, 0, 0, 0, 0, 0, '', 0),
+(85226, 25261, 571, 0, 0, 1, 1, 23031, 0, 2280.96, 5180.91, 11.423, 3.735, 1200, 0, 0, 1, 0, 0, 0, 0, 0, '', 0);
+
+UPDATE `creature_template` SET `flags_extra` = 2, `unit_flags` = 0, `AIName` = 'SmartAI' WHERE `entry` IN (25258, 25259, 25260, 25261);
```

Comment:
```suggestion
UPDATE `creature_template` SET `flags_extra`=`flags_extra`|2, `unit_flags` = 0, `AIName` = 'SmartAI' WHERE `entry` IN (25258, 25259, 25260, 25261);
```
Btw please make sure that these creatures only exist here and never in water, because removing the unit_flag means that you remove `shows swim animation in water`

Reply:
Hi Kitzunu,

The change is as it should, that is the only flag needed by the npcs. ( flags_extra )
About the unit_flags, with ONLY_SWIM on they start randomly to swim in the air, as they never get into water and this fixes it, i think is better this way.